api_fhir_r4/views/fhir/organisation.py

```diff
@@ -35,9 +35,12 @@ class OrganisationViewSet(BaseMultiserializerFHIRView,
     @property
     def serializers(self):
         return {
-            HealthFacilityOrganisationSerializer: (self._hf_queryset(), self._hf_serializer_validator),
-            PolicyHolderOrganisationSerializer: (self._ph_queryset(), self._ph_serializer_validator),
-            InsuranceOrganizationSerializer: (self._io_queryset(), self._io_serializer_validator),
+            HealthFacilityOrganisationSerializer:
+                (self._hf_queryset(), self._hf_serializer_validator, (FHIRApiOrganizationPermissions,)),
+            PolicyHolderOrganisationSerializer:
+                (self._ph_queryset(), self._ph_serializer_validator, (FHIRApiOrganizationPermissions,)),
+            InsuranceOrganizationSerializer:
+                (self._io_queryset(), self._io_serializer_validator, (FHIRApiOrganizationPermissions,)),
```

Comment:
FHIRApiOrganizationPermissions is hard-coded to check policy holder permissions. Can you split it for relevant resources?